akka-persistence/src/main/scala/akka/persistence/journal/AsyncWriteProxy.scala

```diff
@@ -29,10 +29,13 @@ private[persistence] trait AsyncWriteProxy extends AsyncWriteJournal with Stash
 
   override def receive = {
     case SetStore(ref) ⇒
+      println("SetStore(ref) = " + SetStore(ref) + ", " + context.system)
       store = ref
       unstashAll()
       context.become(initialized)
-    case _ ⇒ stash()
+    case x ⇒
+      println("x = " + x)
```

Comment:
Oh, actually will remove here - somewhere on this branch I broke `SharedLeveldbJournalSpec`, this is from investigating it